pkg/base/license.go

```diff
@@ -49,21 +48,22 @@ var licenseTTLMetadata = metric.Metadata{
 	Unit:        metric.Unit_SECONDS,
 }
 
-// LicenseTTL is a metric gauge that measures the number of seconds
-// until the current enterprise license (if any) expires.
-var LicenseTTL = metric.NewGauge(licenseTTLMetadata)
+var AdditionalLicenseTTLMetadata = metric.Metadata{
+	Name:        "seconds_until_license_expiry",
+	Help:        "Seconds until license expiry (0 if no license present or running without enterprise features)",
```

Comment:
Is it intentional that this help string is still referencing enterprise?

Reply:
Hmm this is a good question, I'll check in to see what we want the help string to look like

Reply:
good check, changed the helptext copy